app/templates/views/guidance/index.html

```diff
@@ -24,6 +24,7 @@ <h1 class="heading-large">Guidance</h1>
     <li><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.send_files_by_email') }}">send files by email</a></li>
     <li><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.upload_a_letter') }}">upload a letter</a></li>
     <li><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.schedule_emails_and_text_messages') }}">schedule emails and text messages</a></li>
```

Comment:
Just sneaking this change in here as the PR for this page has been merged already.

```suggestion
    <li><a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('.schedule_emails_and_text_messages') }}">schedule messages</a></li>
```